### PR TITLE
add prediction pipeline 

### PR DESCRIPTION
**Pull request by krajasethupathy**

### CODE REVIEW

File: src/main/scala/com/hbc/data/ml/methods/MLCancelModel.scala [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package com.hbc.data.ml.methods
+
+
+import org.apache.spark.ml.{Pipeline}
+import org.apache.spark.ml.evaluation.Evaluator
+import org.apache.spark.ml.param.ParamMap
+
+
+trait MLCancelModel {
```
Comment (nikolayskovpin) [inline src/main/scala/com/hbc/data/ml/methods/MLCancelModel.scala]: 👍

File: src/main/scala/com/hbc/data/ml/methods/MLCancelClassification.scala [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    "gamma"-> 0.1,
+    "num_round" -> 500,
+    "num_workers" -> 8
+  )
+
+  val cancelModel: XGBoostClassifier = new XGBoostClassifier(xgbParam)
+    .setFeaturesCol("features")
+    .setLabelCol("label")
+
+  override val paramGrid: Array[ParamMap] = new ParamGridBuilder()
```
[the remark this answers is not in the document]
Comment (krajasethupathy) [reply src/main/scala/com/hbc/data/ml/methods/MLCancelClassification.scala]: same params as what?

File: run.sh [hunk 3]
```diff
@@ -17,10 +17,17 @@ STEP_DEFINITION='[{
...
       "--class", "'${CLASS}'",
       "--conf", "spark.yarn.maxAppAttempts=1",
-      "--conf", "spark.dynamicAllocation.enabled=true",
-      "--conf", "spark.executor.memoryOverhead=1800",
-      "--conf", "spark.driver.memoryOverhead=2000",
-      "--conf", "spark.executor.memory=10000M",
+      "--conf", "spark.dynamicAllocation.enabled=false",
+      "--conf", "spark.default.parallelism=200",
+      "--conf", "spark.executor.cores=2",
+      "--conf", "spark.executor.instances=10",
```
[the remark this answers is not in the document]
Comment (krajasethupathy) [reply run.sh]: will change back .. was testing some specs for larger training jobs so i changed this locally

**Comment by krajasethupathy** [comment]
@nikolayskovpin resolved,  + added 1 bug fix

**Pull request merged**